node/src/components/fetcher/fetcher_impls/deploy_fetcher.rs

```diff
@@ -56,7 +59,18 @@ impl ItemFetcher<Deploy> for Fetcher<Deploy> {
         effect_builder: EffectBuilder<REv>,
         id: DeployId,
     ) -> Option<Deploy> {
-        effect_builder.get_stored_deploy(id).await
+        effect_builder
+            .get_stored_transaction(TransactionId::from(id))
+            .await
+            .map(|txn| match txn {
```

Comment:
```suggestion
            .map(|transaction| match transaction {
```

Well, `txn` will always mean DB transaction for me. But surely, not a blocker ;)

Reply:
I have used `txn` pretty much everywhere except `Storage` and I think others will naturally use that abbreviation (or `tx` which I dislike as we generally use that for "transmit" in the context of channels), so I'd rather leave as is.